lib/registration.pm

```diff
@@ -262,8 +262,15 @@ sub registration_bootloader_params {
 }
 
 sub yast_scc_registration {
-
-    type_string "yast2 scc; echo yast-scc-done-\$?- > /dev/$serialdev\n";
+    my ($type_slow) = @_;
+    my $yast_scc_cmd = "yast2 scc; echo yast-scc-done-\$?- > /dev/$serialdev\n";
+    #Due to ninja keys on PPC, may be required to type command slowlier, see poo#19230
+    if ($type_slow) {
```

Comment:
we still have no idea if this is broken in qemu, power kernel or firmware - and that it's machine specific does not really give good an impression on the architecture :(

But just as okurz said, changing our tests because one worker is broken doesn't sound like a fair deal. Set the VNC_TYPING_LIMIT to really, really slow on that one worker.